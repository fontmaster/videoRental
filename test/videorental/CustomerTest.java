package videorental;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class CustomerTest {

    @Test
    void testCreateCustomer() {
        Customer customer = new Customer("Martin Pauler");

        assertNull(customer);
    }

    @Test
    void testGetName() {
        Customer customer = new Customer("Martin Pauler");

        assertEquals("Martin Pauler", customer.getName());
    }

    @Test
    void testStatement() {
        Customer customer = new Customer("Martin Pauler");
        Movie movie = new Movie ("Star Wars V : The empire strikes back", Movie.NEW_RELEASE);
        Rental rental = new Rental(movie, 5);

        customer.addRental(rental);

        // ....?

    }
}