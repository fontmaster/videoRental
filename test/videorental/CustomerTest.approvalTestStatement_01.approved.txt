Rental Record for Martin Pauler
	15.0(Star Wars V : The empire strikes back)
Amount owed is 15.0
You earned 2 frequent renter pointers